components/pages/explorer/QueryActions/Modals/constants.tsx

```diff
@@ -0,0 +1,18 @@
+import { QueryModalsData } from '../types';
+
+export const modalsData: QueryModalsData = {
```

Comment:
this dictionary is WIP for the different Modal types... Not sure it's the best approach as of yet, but given the time constraints, it does the job in a manner easily updated by non-devs.